scripts/install

```diff
@@ -122,22 +122,24 @@ pushd "$install_dir/live"
 	ynh_use_ruby
 	ynh_gem update --system
 	ynh_gem install bundler --no-document
-	ynh_exec_as "$app" "$ynh_ruby_load_path" "$ld_preload" bin/bundle config deployment 'true'
-	ynh_exec_as "$app" "$ynh_ruby_load_path" "$ld_preload" bin/bundle config without 'development test'
-	ynh_exec_as "$app" "$ynh_ruby_load_path" "$ld_preload" bin/bundle config set force_ruby_platform true
-	ynh_exec_as "$app" "$ynh_ruby_load_path" "$ld_preload" bin/bundle install -j$(getconf _NPROCESSORS_ONLN)
+	ynh_exec_as "$app" env $ynh_ruby_load_path $ld_preload bin/bundle config deployment 'true'
+	ynh_exec_as "$app" env $ynh_ruby_load_path $ld_preload bin/bundle config without 'development test'
+	ynh_exec_as "$app" env $ynh_ruby_load_path $ld_preload bin/bundle config set force_ruby_platform true
+	ynh_exec_as "$app" env $ynh_ruby_load_path $ld_preload bin/bundle install -j$(getconf _NPROCESSORS_ONLN)
 
 	# Building assets
 	ynh_use_nodejs
-	corepack enable
-	yarn install
+	env $ynh_node_load_PATH corepack enable
+	ynh_exec_warn_less ynh_exec_as "$app" env $ynh_node_load_PATH corepack enable
```

Comment:
Hmmm why line 132 ? wouldn't line 133 be enough ?

Reply:
If you run as a non-admin user first it'll try to links something in global scope and fall flat on it's face 🤷

Reply:
Yeah indeed.
I got a less impactful fix with an exported env var, in the current pacaging v2 branch, but i like your approach and will test this after packagingv2 is merged onto testing.